lib/scorables/like_given.rb

```diff
@@ -26,9 +26,12 @@ def self.query
           post_actions AS pa
         INNER JOIN posts AS p
           ON p.id = pa.post_id
+        INNER JOIN topics AS t
```

Comment:
This looks a bit weird because the category filter will also add this join. Can you try to integrate it better and make a single join?

Reply:
Ditto for other scorables

Reply:
It can look like the https://github.com/discourse/discourse-gamification/blob/main/lib/scorables/solutions.rb does